The package fails to build due to a collision between two versions of `python3Packages.protobuf`:
- `onnx` brings `python3.10-protobuf-3.20.3`
- `torch` brings `python3.10-protobuf-4.24.2`

The logical thing to do would be to build `onnx` with protobuf 4, but this doesn't work:
```
[ 11%] Building CXX object CMakeFiles/onnx_proto.dir/onnx/onnx-data.pb.cc.o
In file included from /nix/store/6g95lmz80xfif95p86lbpwddas480kdh-protobuf-3.24.2/include/google/protobuf/io/coded_stream.h:130,
                 from /build/source/.setuptools-cmake-build/onnx/onnx-operators-ml.pb.h:24,
                 from /build/source/.setuptools-cmake-build/onnx/onnx-operators-ml.pb.cc:4:
/nix/store/6g95lmz80xfif95p86lbpwddas480kdh-protobuf-3.24.2/include/google/protobuf/stubs/common.h:44:10: fatal error: absl/strings/string_view.h: No such file or directory
   44 | #include "absl/strings/string_view.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /nix/store/6g95lmz80xfif95p86lbpwddas480kdh-protobuf-3.24.2/include/google/protobuf/io/coded_stream.h:130,
                 from /build/source/.setuptools-cmake-build/onnx/onnx-ml.pb.h:24,
                 from /build/source/.setuptools-cmake-build/onnx/onnx-ml.pb.cc:4:
/nix/store/6g95lmz80xfif95p86lbpwddas480kdh-protobuf-3.24.2/include/google/protobuf/stubs/common.h:44:10: fatal error: absl/strings/string_view.h: No such file or directory
   44 | #include "absl/strings/string_view.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /nix/store/6g95lmz80xfif95p86lbpwddas480kdh-protobuf-3.24.2/include/google/protobuf/io/coded_stream.h:130,
                 from /build/source/.setuptools-cmake-build/onnx/onnx-data.pb.h:24,
                 from /build/source/.setuptools-cmake-build/onnx/onnx-data.pb.cc:4:
/nix/store/6g95lmz80xfif95p86lbpwddas480kdh-protobuf-3.24.2/include/google/protobuf/stubs/common.h:44:10: fatal error: absl/strings/string_view.h: No such file or directory
   44 | #include "absl/strings/string_view.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/onnx_proto.dir/build.make:109: CMakeFiles/onnx_proto.dir/onnx/onnx-ml.pb.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/onnx_proto.dir/build.make:123: CMakeFiles/onnx_proto.dir/onnx/onnx-operators-ml.pb.cc.o] Error 1
make[2]: *** [CMakeFiles/onnx_proto.dir/build.make:137: CMakeFiles/onnx_proto.dir/onnx/onnx-data.pb.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:172: CMakeFiles/onnx_proto.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

I am open for help/suggestions on this.

--- a/pkgs/development/python-modules/easyocr/default.nix
+++ b/pkgs/development/python-modules/easyocr/default.nix
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "easyocr";
-  version = "1.7.0";
+  version = "1.7.1";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
@@ -28,7 +28,7 @@ buildPythonPackage rec {
     owner = "JaidedAI";
     repo = "EasyOCR";
     rev = "refs/tags/v${version}";
-    hash = "sha256-01Exz55eTIO/xzdq/dzV+ELkU75hpxe/EbjIqLBA8h0=";
+    hash = "sha256-EiiJ2LJ3uYIvgPd2y25MraV5kTa47JalDR7SLbkM9UI=";
   };
 
   postPatch = ''

